colossalai/utils/model/utils.py

```diff
@@ -80,6 +80,10 @@ def __exit__(self, exc_type, exc_value, traceback):
             torch.set_default_dtype(self._old_default_dtype)
 
         def _disable_class(cls):
+            if not hasattr(cls, '_old_init'):
+                raise AttributeError(
+                    f"_old_init is not found in the {cls.__name__}, please make sure that you have this class imported before entering the context."
```

Comment:
make sure import InsertPostInitMethodToModuleSubClasses?

Reply:
make sure cls is imported

Reply:
I can make it clearer

Reply:
done